routers/init.go

```diff
@@ -107,7 +107,7 @@ func InitWebInstallPage(ctx context.Context) {
 }
 
 // InitWebInstalled is for global installed configuration.
-func InitWebInstalled(ctx context.Context) {
+func InitWebInstalled(ctx context.Context, extraDBPrepare func() error) {
```

Comment:
It's really hacky, I do not think it's healthy for the code base.

Reply:
Yes, it is. For production, all the data are there in the database, for test env, data are loaded from fixtures after database initlized. I couldn't find a quick way to satisfy two situations. Just leave it for another PR to refactor it.

Reply:
I think you can add the function call to the test code only, instead of making production code more complex.

Reply:
But then we need to split `InitWebInstalled` into some small functions which just invoked twice. Is this really necessary?